easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.5-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+# initially generated with eb ABINIT-7.10.4-intel-2015a.eb --try-software-version=7.10.5
+# --try-toolchain=intel,2016.02-GCC-4.9, then added intel mpi and fft support
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = "7.10.5"
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = []
```

Comment:
since in this case fetching new versions with --try-version should work (not always the case, e.g. in QuantumESPRESSO we cannot antecipate the url of new versions), opted to remove the line instead of adding the checksum for this particular version, to keep it more general, do let me know if adding the specific checksum is preferred (e.g., if the version is changed from the command line, does eb automatically skip the checksum? if so, I'll add it)